pkg/kubectl/rollback.go

```diff
@@ -108,7 +108,7 @@ func (r *DeploymentRollbacker) Rollback(obj runtime.Object, updatedAnnotations m
 // watchRollbackEvent watches for rollback events and returns rollback result
 func watchRollbackEvent(w watch.Interface) string {
 	signals := make(chan os.Signal, 1)
-	signal.Notify(signals, os.Interrupt, os.Kill, syscall.SIGTERM)
+	signal.Notify(signals, os.Interrupt, os.Kill, unix.SIGTERM)
```

Comment:
I have only tested this with go 1.8.3 but I'd guess this should work with every version, as the underlying type of both `syscall.Signal` and `unix.Signal` is `int`.